module/Checker/src/Checker/Service/Checker.php

```diff
@@ -0,0 +1,210 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: stefan
+ * Date: 22-12-14
+ * Time: 18:07
+ */
+namespace Checker\Service;
+
+use Application\Service\AbstractService;
+use \Database\Model\SubDecision\Foundation;
+use \Database\Model\Meeting;
+use \Checker\Model\Error;
+use Zend\Mail\Message;
+
+class Checker extends AbstractService
+{
+    /**
+     * Does a full check on each meeting, checking that after each meeting no database violation occur
+     */
+    public function check()
+    {
+        $meetingService = $this->getServiceManager()->get('checker_service_meeting');
+        $meetings = $meetingService->getAllMeetings();
+
+        $message = '';
+        foreach ($meetings as $meeting) {
+            $errors = array_merge(
+                $this->checkBudgetOrganExists($meeting),
+                $this->checkMembersHaveRoleButNotInOrgan($meeting),
+                $this->checkMembersInNotExistingOrgans($meeting),
+                $this->checkMembersExpiredButStillInOrgan($meeting),
+                $this->checkOrganMeetingType($meeting)
+            );
+
+            $message .= $this->handleErrors($meeting, $errors);
+        }
+
+        $this->sendMail($message);
+    }
+
+    /**
+     * Makes sure that the errors are handled correctly
+     *
+     * @param \Database\Model\Meeting $meeting Meeting for which this errors hold
+     * @param array $errors
+     */
+    private function handleErrors(\Database\Model\Meeting $meeting, array $errors)
+    {
+        // At this moment only write to output.
+        $body =  'Errors after meeting ' . $meeting->getNumber() . ' hold at '
+            . $meeting->getDate()->format('Y-m-d') . "\n";
+
+        foreach ($errors as $error) {
+            $body.= $error->asText() . "\n";
+        }
+
+        $body .= "\n";
+        return $body;
+
+    }
+
+    /**
+     * Send a mail with the detected errors to the secretary
+     *
+     * @param $body
+     */
+    private function sendMail($body)
+    {
+        $transport = $this->getServiceManager()->get('checker_mail_transport');
+
+        $message = new Message();
+        $message->addTo('secr@gewis.nl')
```

Comment:
Doel je op het email adres die nog naar een config file moet? Of om het feit dat het handelen van de errors eigenlijk niet abstract genoeg is?

Dat eerste is makkelijk te veranderen